studio-docs/source/api-keys.mdx

```diff
@@ -10,22 +10,29 @@ Every application that sends data to Apollo must use an **API key** to do so. Ap
 
 ## Graph API keys
 
-A graph API key provides **full** access to a **single graph** in Apollo Studio. A GraphQL server (such as Apollo Server) can use a graph API key to register its schema and push metrics to Studio.
+A graph API key provides access to a **single graph** in Apollo Studio. A GraphQL server (such as Apollo Server) can use a graph API key to register its schema and push metrics to Studio.
 
 Create a unique graph API key for each non-development system that communicates with Studio. Doing so enables you to revoke access to a single system without affecting others.
 
 <ObtainGraphApiKey />
 
+### Setting permissions
+
+Unless you have an Enterprise plan, every graph API key provides **full access** to its associated graph.
+
+If you have an Enterprise plan, you can [assign a member role](/org/members/#member-roles) to each graph API key you create. If you do, the API key's permissions are limited to that role's permissions.
```

Comment:
Anchor needs to be updated. Although — do we do anchor redirects (or specify specific anchors on headings that have changed) in our docs these days?

Reply:
I don't know. Is suspect we probably don't do a very good job with anchor redirects, even if that is technically possible.